### DISCUSSION TITLE
Updating some packages that are out of date

### DISCUSSION THREAD

**Discussion by Rhialto** [Homebrew/discussions#2779] [Everyday usage]
Hi! I'm a fresh homebrew user (I had to switch from MacPorts for $REASONS). I noticed a few packages that could use an update / improvement:

- x3270, which doesn't actually install the x3270 binary, which is pretty much the main goal of the upstream suite-3270. x3270 compiles and works fine if you have XQuartz installed. It doesn't even have any other dependencies.
- hercules, which, at version 3.13 has been unmaintained for years now. There is a well-maintained version 4.4.1 (which is a fork of a fork of version 3) with home page https://sdl-hercules-390.github.io/html/ and releases at https://github.com/SDL-Hercules-390/hyperion/releases/ .
- simh, which unfortunately does not do tags or releases, but the version "v3.11-1" is not the latest version. The trunk (at https://github.com/simh/simh) contains version 4 which is regularly updated, where version 3 is only changed very sparingly ever since Mark Pizzolato started version 4 in 2012. I'm not even sure what branch v3.* are on. Maybe it makes sense to have 2 packages for v3 and v4?

**Comment by carlocab** [comment]
Hi there -- thanks for the info. Can I interest you in opening a pull request for any of these formulae? It's not that difficult once you get the hang of it, and we're happy to give you lots of help if you get stuck.

If you're up for it, this would be a good place to start: https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md#contributing-to-homebrew

**Comment by Rhialto** [comment]
Thanks for the encouragement. I won't have so much time for it, since the Mac in question is from $WORK so my time to spend on it is limited. I'm also an active contributor to [pkgsrc](https://www.pkgsrc.org) but on the other hand it gives me experience with precisely these packages, since I keep them up to date there.
